eclipse/ide-target-platform/category.xml

```diff
@@ -24,8 +24,8 @@
    <bundle id="org.hamcrest" version="1.1.0.v20090501071000"/>
    <bundle id="org.mortbay.jetty.server" version="6.1.26"/>
    <bundle id="org.mortbay.jetty.util" version="6.1.26"/>
-   <bundle id="com.google.guava" version="20.0"/>
-   <bundle id="com.google.gson" version="2.7"/>
+   <bundle id="com.google.guava" version="20.0.0"/>
```

Comment:
You're right. The OSGi version and the Maven version are different, like gson and freemarker. You can check the OSGi version of guava from its `MANIFEST.MF`.